api/routes/affiliations/get.js

```diff
@@ -0,0 +1,20 @@
+const { raw: knex } = require('../../db');
+const { can } = require('../../middleware');
+
+module.exports = (app) => {
+  app.get('/states/:stateId/affiliations', can('view-users'), async (request, response) => {
```

Comment:
@thetif Does it make sense to create records in the 'auth_activities' table for `{add,view,edit,delete}-affiliations`, and apply them here with `can`?

Reply:
Another question: Would it make more sense to nest this under the `/auth` path, since it deals with authorization?

Reply:
Yes, I think you should add those to the auth_activities table. You might want to remove the *-roles in there too since we won't need those any more.

Is it under users now? I think that's a good place for it because that's where the old user management stuff was.